pipeline/lib/pipeline.ml

```diff
@@ -210,17 +212,17 @@ let exists ~conninfo repository =
   db#finish;
   exists
 
-let process_pipeline ~ocluster ~conninfo ~sources () =
+let process_pipeline ~config ~ocluster ~conninfo ~sources () =
   Current.list_iter ~collapse_key:"pipeline"
     (module Repository)
     (fun repo ->
       let* repository = repo in
       if exists ~conninfo repository
       then Current.ignore_value repo
-      else pipeline ~ocluster ~conninfo repository)
+      else pipeline ~config ~ocluster ~conninfo repository)
     (repositories sources)
 
-let v ~current_config ~server:mode ~sources conninfo () =
+let v ~config ~current_config ~server:mode ~sources conninfo () =
```

Comment:
Can we rename `config`? Having both `config` and `current_config` as parameters looks a bit confusing :)

Reply:
Indeed! `current_config` is not actually used so it was removed :)